### PR TITLE
Definition snippets

### PR DESCRIPTION
**Pull request by soulshake**
## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by soulshake** [comment]
[Definition snippets in the docs](https://trello.com/c/QPaePj9T/1212-definition-snippets-in-the-docs)

File: _includes/definitions/resource.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Resources are like services, but are external to your application and which your application communicates with over a network.
+Examples of typical resources used with Convox are data stores like RDS or redis, mailservers, and so on.
```
Comment (mattmanning) [inline _includes/definitions/resource.md]: Should there be a newline here?

File: _includes/definitions/resource.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+Resources are typically stateful services.
+
+Resources correspond closely to Heroku's notion of [_backing services_](https://devcenter.heroku.com/articles/development-configuration#backing-services).
```
Comment (mattmanning) [inline _includes/definitions/resource.md]: Can we link to https://12factor.net/backing-services and reference 12factor instead of Heroku?

Comment (soulshake) [inline _includes/definitions/resource.md]: Oops, yes, will fix

**Comment by soulshake** [comment]
Fixed Heroku -> 12-factor *for real* this time

**Pull request merged**